modules/installation-osp-enabling-swift.adoc

```diff
@@ -7,14 +7,16 @@
 [id="installation-osp-enabling-swift_{context}"]
 = Enabling Swift on OpenStack
 
-{product-title} on {rh-openstack-first} uses https://docs.openstack.org/security-guide/object-storage.html[OpenStack Object Storage (Swift)] to store and serve user configuration files.
+If link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html-single/storage_guide/index#ch-manage-containers[OpenStack Object Storage (Swift)] is available, {product-title} on {rh-openstack-first} uses it to store and serve user configuration files. If it is unavailable, the installation program relies on the OpenStack Image service (Glance).
```

Comment:
We're unconditionally using glance for storing the bootstrap ignition file.

Reply:
The PR you linked in your other comment includes this language:

>If Swift is available in the cloud where the installation is being performed, it is used as the default backend for the OpenShift image registry.

Is that not accurate anymore?

Reply:
@mandre ^^^

Reply:
Starting from 4.4:
- we use glance to store the bootstrap ignition file (used to be swift)
- we continue to use swift for the image registry if available, and fallback to cinder if not the case

Reply:
@mandre Do you think it's worth saying that Swift or Cinder are requirements, then? I'm seeing some 'minimal' OSP deployments in their docs that don't even require Cinder, but I'm not sure how common this actually is in the real world.

Reply:
Yes, we should probably mention it somewhere, although deployment without object _and_ block storage must be quite rare :man_shrugging: